tests/tests/test-democracy.ts

```diff
@@ -367,7 +367,7 @@ describeDevMoonbeam("Democracy - forget notePreimage", (context) => {
         Standard: { balance: VOTE_AMOUNT, vote: { aye: true, conviction: 1 } },
       })
     );
-    expect(eventsVote[3].toHuman().method).to.eq("ExtrinsicSuccess");
+    expect(eventsVote[4].toHuman().method).to.eq("ExtrinsicSuccess");
```

Comment:
This was my change that "fixed" the test, @JoshOrndorff . I don't know if that's correct or not, so please review :)

Reply:
Yes, this makes sense/